clients/arweave_ipfs_s3.go

```diff
@@ -69,7 +53,8 @@ func DownloadDStorageFromGatewayList(u string, requestID string) (io.ReadCloser,
 		}
 	}
 
-	for _, gateway := range gateways {
+	for i := retry; i < len(gateways); i++ {
```

Comment:
Why do we skip the first gateway when we retry, then the first two on the second retry??